packages/gestalt/src/contexts/DefaultLabelProvider.js

```diff
@@ -59,14 +59,14 @@ export const fallbackLabels: DefaultLabelContextType = {
   Popover: {
     accessibilityDismissButtonLabel: 'Close popover',
   },
-  Sheet: {
-    accessibilityDismissButtonLabel: 'Close sheet',
+  OverlayPanel: {
+    accessibilityDismissButtonLabel: 'Close overlaypanel',
```

Comment:
these need spaces

Reply:
I swear I thought about this space... guess a spirit came and took it out